notebook.ipynb

```diff
@@ -2,7 +2,15 @@
  "cells": [
```

Comment:
<ul><li>Need to spell out CLI acronym on first instance.</li><li>Maybe link to <a href="https://github.com/icenet-ai/icenet-pipeline?tab=readme-ov-file#configuration" target="_blank">https://github.com/icenet-ai/icenet-pipeline?tab=readme-ov-file#configuration</a> for the configuration details</li><li>Need a link to what Airflow is. Assuming that you mean Apache Airflow.</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/eds-book-gallery/67a1e320-7c47-4ea9-8df8-e868326bc90b/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>

Reply:
<p>* Added acronym</p><p>* Updated link as suggested.</p><p>* Added link to Apache Airflow.</p>